dashboard/db/schema.rb

```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```

Comment:
Presumably the `options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci"` changes fit into this category?